build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -26,3 +26,5 @@
 LARGE = [
     tf_models.MINILM_L12_H384_UNCASED_INT32_SEQLEN128,
 ]
+
+ALL = SMALL + LARGE
```

Comment:
Is it used anywhere? I can't find the usage in `build_tools/python/benchmark_suites`.

Reply:
It's used in `build_tools/python/e2e_test_artifacts/artifacts.py`

Reply:
In that case, this change should be in https://github.com/iree-org/iree/pull/10901

Reply:
Nice catch. Done.